src/main/utils/fact-api.ts

```diff
@@ -0,0 +1,17 @@
+import axios, { AxiosResponse, AxiosError } from 'axios';
+import config from 'config';
+
+export class FactApi {
+  static search(query: string): Promise<[]> {
```

Comment:
don't make this static, you can't mock it

Reply:
i didnt wanna initialise the class so i made it an static method. for unit test, i did spy on the function and returned the value i wanted ecpect it to return

Reply:
Feels like we need some form of dependency injection